frontend/src/components/cluster/Charts.tsx

```diff
@@ -19,7 +19,7 @@ export function MemoryCircularChart(props: ResourceCircularChartProps) {
   }
 
   function memoryAvailableGetter(item: Node | Pod) {
-    return parseRam(item.status!.capacity.memory) / TO_GB;
+    return parseRam(item.status!.capacity?.memory) / TO_GB;
```

Comment:
might as well remove the `!` here `item.status?.capacity?.memory`

Reply:
I will change this.